test/unit/collect_active_set.c

```diff
@@ -79,22 +79,26 @@ int main(void)
 
     /* A total of npes tests are performed, where the active set in each test
      * includes PEs i..npes-1 and each PE contributes PE ID elements */
-    for (i = 0; i <= me; i++) {
+    shmem_team_t new_team;
+    for (i = 0; i < npes; i++) {
```

Comment:
Since all of the PEs in the parent team need to call split_strided,  we will not be able to use the  v1.5 collectives with the previous for loop (i.e. i <= me). With the for loop iterating across i >= me, a PE from the parent team will always drop out of the loop causing a hang when it comes time to call split_strided on the subsequent iteration.

However, I was able to wrap the necessary code with #ifdef ENABLE_DEPRECATED_TESTS.